app/register/page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test';
+
+const goodUser = {
+  email: 'test1@email.com',
```

Comment:
Shashi's gonna ask you to alphabetize this I'd imagine...

Reply:
I definitely could alphabetize it, but for this particular process, I'd prefer to keep it in this order since this is how the logic works for registering a user.

Please let me know your thoughts @shashilo